src/pages/pass-finalisation/pass-finalisation.ts

```diff
@@ -119,6 +130,24 @@ export class PassFinalisationPage extends PracticeableBasePageComponent {
         select(getPassCertificateNumber),
         tap(val => this.form.controls['passCertificateNumberCtrl'].setValue(val)),
       ),
+      transmission$: currentTest$.pipe(
+        select(getVehicleDetails),
+        select(getGearboxCategory),
+      ),
+      transmissionAutomaticRadioChecked$: currentTest$.pipe(
```

Comment:
Can you not drive the checked state purely from `transmission$` by making the value on the `input` match up with the materialised value? You're always just setting a value on the form control.